exec/pipeline/financials/pull_data.py

```diff
@@ -0,0 +1,74 @@
+from prefect import flow, task
+from pydantic import BaseModel
+import requests
+import instructor
+from pydantic import BaseModel, validator
+from openai import OpenAI
+import tempfile
+from decimal import Decimal
+from openai import OpenAI
+
+import PyPDF2
+
+
+class CurrentAssets(BaseModel):
+    cash_and_cash_equivalents: Decimal
+    marketable_securities: Decimal
+    accounts_receivable_net: Decimal
+    vendor_non_trade_receivables: Decimal
+    other_current_assets: Decimal
+    total_current_assets: Decimal
+
+    # @validator("total_current_assets")
+    # def total_current_assets_is_sum_of_all_assets(cls, v):
+    #     return v == (
+    #         cls.cash_and_cash_equivalents
+    #         + cls.marketable_securities
+    #         + cls.accounts_receivable_net
+    #         + cls.vendor_non_trade_receivables
+    #         + cls.other_current_assets
+    #     )
```

Comment:
Are you intentionally going to keep this for now or are you deleting it?

Reply:
yea, might as well just delete